sdk/objectanchors/Azure.MixedReality.ObjectAnchors.Conversion/src/NotSupportedAssetFileTypeException.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Runtime.Serialization;
+
+namespace Azure.MixedReality.ObjectAnchors.Conversion
+{
+    /// <summary>
+    /// An Exception thrown during an attempt to provide an unsupported asset file type in an asset conversion operation
+    /// </summary>
+    public class NotSupportedAssetFileTypeException : Exception, ISerializable
```

Comment:
I think ISerializable is redundant here as Exception implements it.

Reply:
@tg-msft thoughts here? this was based on your feedback

Reply:
This is just a nit - feel free to ignore.